src/collar/oc_capture.lsl

```diff
@@ -22,288 +22,359 @@ integer CMD_SAFEWORD = 510;
+        llMessageLinked(LINK_DIALOG, NOTIFY, "0" + "\n\n%WEARERNAME% is at your mercy.\n\nNOTE: During capture RP %WEARERNAME% cannot refuse your teleport offers and you will keep full control. Type \"/%CHANNEL% %PREFIX% grab\" to attach a leash or \"/%CHANNEL% %PREFIX% capture release\" to relinquish capture access to %WEARERNAME%'s %DEVICETYPE%.\n\nHave fun! For basic instructions click [here].\n", sCaptorID);
         g_sTempOwnerID = sCaptorID;
-        saveTempOwners();
+        SaveTempOwners();
         llSetTimerEvent(0.0);
     }
 }
 
-UserCommand(integer iNum, string sStr, key kID, integer remenu) {
-    string sStrLower=llToLower(sStr);
-    if (llSubStringIndex(sStr,"capture TempOwner") == 0){
-        string sCaptorID = llGetSubString(sStr,llSubStringIndex(sStr,"~")+1,-1);
-        if (iNum==CMD_OWNER || iNum==CMD_TRUSTED || iNum==CMD_GROUP) { //do nothing, owners get their own menu but cannot capture
+UserCommand(integer iNum, string sStr, key kID, integer bRemenu) {
+    string sStrLower = llToLower(sStr);
+
+    if (llSubStringIndex(sStr, "capture TempOwner") == 0) {
+        string sCaptorID = llGetSubString(sStr, llSubStringIndex(sStr, "~") + 1, -1);
+        if (iNum == CMD_OWNER || iNum == CMD_TRUSTED || iNum == CMD_GROUP) { // Do nothing, owners get their own menu but cannot capture
+        } else {
+            Dialog(kID, "\nYou can try to capture %WEARERNAME%.\n\nReady for that?", ["Yes","No"], [], 0, iNum, "ConfirmCaptureMenu", sCaptorID);
         }
-        else Dialog(kID, "\nYou can try to capture %WEARERNAME%.\n\nReady for that?", ["Yes","No"], [], 0, iNum, "ConfirmCaptureMenu", sCaptorID);
-    }
-    else if (sStrLower == "capture" || sStrLower == "menu capture") {
-        if  (iNum!=CMD_OWNER && iNum != CMD_WEARER) {
-            if (g_iCaptureOn) Dialog(kID, "\nYou can try to capture %WEARERNAME%.\n\nReady for that?", ["Yes","No"], [], 0, iNum, "ConfirmCaptureMenu", kID);
-            else llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"%NOACCESS%",kID);//Notify(kID,g_sAuthError, FALSE);
-        } else CaptureMenu(kID, iNum); // an authorized user requested the plugin menu by typing the menus chat command
-    }
-    else if (iNum!=CMD_OWNER && iNum != CMD_WEARER){
-        //silent fail, no need to do anything more in this case
-    }
-    else if (llSubStringIndex(sStrLower,"capture")==0) {
-        if (g_sTempOwnerID != "" && kID==g_kWearer) {
-            llMessageLinked(LINK_DIALOG,NOTIFY,"0"+"%NOACCESS%",g_kWearer);
+    } else if (sStrLower == "capture" || sStrLower == "menu capture") {
+        if  (iNum != CMD_OWNER && iNum != CMD_WEARER) {
+            if (g_bCaptureOn) {
+                Dialog(kID, "\nYou can try to capture %WEARERNAME%.\n\nReady for that?", ["Yes", "No"], [], 0, iNum, "ConfirmCaptureMenu", kID);
+            } else {
+                llMessageLinked(LINK_DIALOG, NOTIFY, "0" + "%NOACCESS%", kID); // Notify(kID, g_sAuthError, FALSE);
```

Comment:
After testing your version. I am getting access denied messages in this script. To reproduce. Turn public mode on and also turn on capture.  Then have someone else touch and they'll get a access denied message at least twice.